spec/test/test-panel-spec.js

```diff
@@ -0,0 +1,71 @@
+/** @babel */
+/* eslint-env jasmine */
+
+import TestPanel from './../../lib/test/test-panel'
+
+describe('test panel', () => {
+  let testPanel
+
+  beforeEach(() => {
+    testPanel = new TestPanel()
+  })
+
+  describe('make link', () => {
```

Comment:
add two more tests:

_pre- and suffix_
input string: `"prefix foo/bar.go:23 suffix"`
output elements:
* SPAN with text `"prefix "`
* A with text `"foo/bar.go:23"`
* SPAN with text `" suffix"`

_multi line text_
input string: `"--- FAIL: TestFail (0.00s)\n\tbar_test.go:23: Error!\nFAIL"`
output elements:
* SPAN with text `"--- FAIL: TestFail (0.00s)\n\t"`
* A with text `"bar_test.go:23"`
* SPAN with text `": Error!\nFAIL"`